<launch>
  <node name="orion_state" pkg="orion_state" type="orion_state.py" output="screen">
    <param name="rate" value="20"/>

    <remap from="/aio/in/gearbox_temp_left" to="/ugr/can/lv/processed/state_bpri1"/>
    <remap from="/aio/in/gearbox_temp_right" to="/ugr/can/lv/processed/state_bpri2"/>
    <remap from="/aio/in/front_bp" to="/ugr/can/lv/processed/state_bpri3"/>
    <remap from="/aio/in/rear_bp" to="/ugr/can/lv/processed/state_bpri4"/>
    <remap from="/aio/in/air_pressure1" to="/ugr/can/lv/processed/state_bpri5"/>
    <remap from="/aio/in/air_pressure2" to="/ugr/can/lv/processed/state_bpri6"/>
  </node>
</launch>